ruby/vampires.rb

```diff
@@ -0,0 +1,106 @@
+#Now we ask how many applicants need to be interviewed!
+i = 0
+n = 0
+
+while i == 0
+	puts "How many employees will you be interviewing?"
+	i = gets.chomp.to_i

```

Comment:
Think about naming variables with what they represent, opposed to letters and abbreviations in order to increase readability.